crates/re_types_builder/src/codegen/rust.rs

```diff
@@ -693,8 +693,13 @@ fn quote_trait_impls_from_obj(
             quote! {
                 #into_cow
 
+                impl #name {
+                }
```

Comment:
?

Reply:
Good catch. Had added a different non-trait helper in an intermediate revision.